src/Module.Server/Common/Models/CrpgAgentApplyDamageModel.cs

```diff
@@ -52,9 +53,16 @@ public CrpgAgentApplyDamageModel(CrpgConstants constants)
 
         if (weapon.IsEmpty)
         {
-            // Increase fist damage with strength.
+            // Increase fist damage with strength and glove armor.
+            // Alternate attack if is weapon empty is a kick.
             int strengthSkill = GetSkillValue(attackInformation.AttackerAgentOrigin, CrpgSkills.Strength);
-            return finalDamage * (1 + 0.03f * strengthSkill);
+            int glovearmor = GetGloveArmor(attackInformation.AttackerAgentOrigin);
+            if (collisionData.IsAlternativeAttack)
+            {
+                return (finalDamage * 0.75f) * (1 + 0.02f * strengthSkill);
+            }
+
+            return (finalDamage * 0.75f) * (1 + (0.02f * strengthSkill) + (0.04f * glovearmor));
         }
```

Comment:
you should use collision_data to check if Is_Charge.

Reply:
for more clarity you can do 
` 
```
if (weapon.IsEmpty)
{
  finaldamage *= 0.75 * (1 + 0.02f * strengthSkill);

  if (collisionData.IsAlternativeAttack) // Kick
  {
    return finaldamage;
  }

  int glovearmor = GetGloveArmor(attackInformation.AttackerAgentOrigin);
  return finaldamage * (0.04f * glovearmor));
}
```
        
 Also good code is self commenting. By that i mean that if the code is well writtent , it can be read like a story , without further comment.
 Comment should be reserved for places where the reader would have the reaction "uh? What is that?"
 
 Do not forget to add a check for is_charge